integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -546,6 +548,18 @@ class CarbonSqlParser()
     colGrpNames.toString()
   }
 
+  protected def getTableBlockSize(tableProperties: Map[String, String]): Integer = {
+    var tableBlockSize: Integer = 0
+    if (tableProperties.get(CarbonCommonConstants.TABLE_BLOCKSIZE).isDefined) {
+      val blockSizeStr: String = tableProperties.get(CarbonCommonConstants.TABLE_BLOCKSIZE).get
+      try {
+        tableBlockSize = Integer.parseInt(blockSizeStr)
+      } catch {
+        case e: NumberFormatException => tableBlockSize = 0
```

Comment:
better assign default value. so that system will be in a consistent state always